woocommerce/woocommerce-android
# Product creation using package photo: Media picker dialog

This PR adds a product picker dialog implemented in Compose, which will launch the media picker library (next PR) for selecting an image. 

**To test:**
1. Go to Products
2. Tap on + FAB
3. Select the AI option
4. Tap on Use package photo button
5. Notice a media picker dialog with 3 options appears

<img src="https://github.com/woocommerce/woocommerce-android/assets/1522856/be11efaf-c3f8-4005-ad37-ec6bed0e1d6c" width="400" />
